trunk/SUAVE/Methods/Flight_Dynamics/Dynamic_Stability/single_point_aero_derivatives.py

```diff
@@ -0,0 +1,188 @@
+## @ingroup Methods-Flight_Dynamics-Dynamic_Stability
+# single_point_aero_derivatives.py
+# 
+# Created:   Aug 2021, R. Erhard
+# Modified: 
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Data
+from SUAVE.Plots.Mission_Plots import *  
+
+import numpy as np 
+from copy import deepcopy
+
+## @ingroup Methods-Flight_Dynamics-Dynamic_Stability
+def single_point_aero_derivatives(analyses,vehicle, state, h=0.01): 
```

Comment:
Instead of calling this function separately. Why not add it as post process step? Like these:
https://github.com/suavecode/SUAVE/blob/develop/trunk/SUAVE/Analyses/Mission/Segments/Cruise/Constant_Speed_Constant_Altitude.py#L143